### PR TITLE
Fetch token prices excess network calls / rate limited fix

### PR DESCRIPTION
**Pull request by kev1n-peters**
We should fetch the token prices once when mounted and refresh them periodically. Doing this instead of fetching them on demand in multiple components helps prevent us from getting rate limited and making redundant network calls.

### CODE REVIEW

File: wormhole-connect/src/views/v2/Redeem/TransactionDetails/index.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ const TransactionDetails = () => {
             {amount} {sourceTokenConfig.symbol}
           </Typography>
           <Typography color={theme.palette.text.secondary} fontSize={14}>
-            {isFetchingTokenPrices ? (
+            {tokenPrices.isFetching ? (
```
Comment (artursapek) [inline wormhole-connect/src/views/v2/Redeem/TransactionDetails/index.tsx:96]: I don't think any code actually sets this `isFetching` boolean to true anymore. From what I can tell it's set using `setFetchingPrices` in `stores/tokenPrices.ts` but I don't see that function called anywhere. Maybe we can default `isFetching` to `true` inside the `initialState` value?

File: wormhole-connect/src/views/v2/Bridge/index.tsx [hunk 2]
```diff
@@ -184,6 +185,9 @@ const Bridge = () => {
   // Call to initiate transfer inputs validations
   useValidate();
 
+  // Fetch token prices
+  useFetchTokenPrices();
```
Comment (artursapek) [inline wormhole-connect/src/views/v2/Bridge/index.tsx:189]: Nice improvement calling this hook only in the top level here.

File: wormhole-connect/src/views/v2/Redeem/TransactionDetails/index.tsx (hunk 1 above)
Comment (kev1n-peters) [reply wormhole-connect/src/views/v2/Redeem/TransactionDetails/index.tsx:96]: I decided to call `dispatch(setFetchingPrices())` once in `fetchTokenPrices` instead, wdyt

Comment (artursapek) [reply wormhole-connect/src/views/v2/Redeem/TransactionDetails/index.tsx:96]: yeah I think once at the beginning makes sense!

**Review by artursapek — APPROVED** [review]

**Pull request merged**